ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -371,7 +371,7 @@ Item {
 
                 if (link.startsWith('//')) {
                   let pk = link.replace("//", "");
-                  profileClick(chatsModel.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk))
+                  profileClick(chatsModel.status.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk))
```

Comment:
From QML, it's better to call the View proc and let the View proc call the model or controller.